packs/data/feat-effects.db/effect-rage.json

```
@@ -43,11 +43,9 @@
                 "value": "@actor.flags.pf2e.rageTempHP"
             },
             {
-                "key": "ActiveEffectLike",
-                "mode": "override",
-                "path": "flags.pf2e.rollOptions.all.rage",
-                "priority": 10,
-                "value": true
+                "domain": "all",
+                "key": "RollOptions",
```

```suggestion
                "key": "RollOption",
```
Also is this entire RE even still necessary at this point?

---

Will people who haven't updated their barbarians lose functionality if it's removed?

---

Possibly? Those would be some old barbarians.